frontend/plan/src/components/schedule/Stats.js

```diff
@@ -57,22 +51,22 @@ class Stats extends Component {
             if (str) {
                 const course = str.substring(0, str.indexOf("-", str.indexOf("-") + 1)); // finds course (irrespective of section)
                 if (course in courseDifficulties) {
-                    courseDifficulties[course] += (section.difficulty ? section.difficulty : 2.0);
+                    courseDifficulties[course] += (section.difficulty ? section.difficulty : 2.5);
                     courseWorkloads[course] += (section.work_required
-                        ? section.work_required : 2.0);
```

Comment:
I think the better way to do this would be to not add to the sum when the field isn't there? From some back of the napkin math, doesn't seem like either 2.0 or 2.5 does what we want it to do (keep the avg the same).